app/src/main/java/com/ayush/weatherapp/main/MainActivity.java

```diff
@@ -32,6 +38,11 @@
     setActionBar();
 
     setNavigationView();
+
+    presenter = new MainPresenter(this);
+    presenter.setView(this);
+
+    presenter.apiCall();
```

Comment:
presenter.fetchWeatherDetail would be nice

Reply:
:ok: